extensions/amp-accordion/1.0/README.md

```diff
@@ -0,0 +1,622 @@
+
+```jsx
+    <BentoAccordion ref={ref}>
+      <BentoAccordionSection
+        id="section1"
+        key={1}
+        onExpandStateChange={
+          (expanded) => expanded ? alert('section1 expanded') : alert('section1 collapsed')
+        }>
+        <BentoAccordionHeader><h1>Section 1</h1></BentoAccordionHeader>
+        <BentoAccordionContent>Content 1</BentoAccordionContent>
+      </BentoAccordionSection>
+
+      <BentoAccordionSection
+        id="section2"
+        key={2}
+        onExpandStateChange={
+          (expanded) => expanded ? alert('section2 expanded') : alert('section2 collapsed'}
+        }>
+        <BentoAccordionHeader><h1>Section 2</h1></BentoAccordionHeader>
+        <BentoAccordionContent>Content 2</BentoAccordionContent>
+      </BentoAccordionSection>
+
+      <BentoAccordionSection
+        id="section3"
+        key={3}
+        onExpandStateChange={
+          (expanded) => expanded ? alert('section3 expanded') : alert('section3 collapsed'}
+        }>
+        <BentoAccordionHeader><h1>Section 3</h1></BentoAccordionHeader>
+        <BentoAccordionContent>Content 3</BentoAccordionContent>
+      </BentoAccordionSection>
+    </BentoAccordion>
+```
+
+#### Layout and style
+
+**Container type**
+
+The `BentoAccordion` component has a defined layout size type. To ensure the component renders correctly, be sure to apply a size to the component and its immediate children via a desired CSS layout (such as one defined with `height`, `width`, `aspect-ratio`, or other such properties). These can be applied inline:
+
+```jsx
+<BentoAccordion style={{width: '300px', height: '100px'}}>
+  ...
+</BentoAccordion>
+```
+
+Or via `className`:
+
+```jsx
+<BentoAccordion className='custom-styles'>
+  ...
+</BentoAccordion>
+```
+
+```css
+.custom-styles {
+  background-color: red;
+}
+```
+
+#### Props
+
+##### BentoAccordion
+
+###### animate
+
+If true, then uses "roll-down" / "roll-up" animation during the expansion and collapse of each section
+Default: `false`
+
+###### expandSingleSection
+
+If true, then expanding 1 section will automatically collapse all other sections:
+Default: `false`
+
+##### BentoAccordionSection
+
+###### animate
+
+If true, then uses "roll-down" / "roll-up" animation during the expansion and collapse the section
+Default: `false`
+
+###### expanded
+
+If true, expands the section.
+Default: `false`
+
+###### onExpandStateChange
+
+```typescript
+(expanded: boolean): void
+```
+
+Callback to listen for expand state changes. Takes a boolean flag as parameter indicating whether the section was just expanded (`false` indicates it was collapsed)
+
+##### BentoAccordionHeader
+
+##### BentoAccordionContent
+
+#### Styling
+
+You may use the `BentoAccordion` element selector to style the carousel freely.
+
+## Version notes
+
+The experimental `1.0` version of `amp-accordion` does not support session states. It behaves as if the `disable-session-states` attribute is always applied.
```

Comment:
Sorry if this was unclear earlier -- The whole `## Version notes` section can be removed

Reply:
removed the rest - left the header